webpack.config.js

```diff
@@ -349,7 +349,10 @@ const styles = {
 		const entries = {};
 		const dir = './assets/css/src';
 		fs.readdirSync( dir ).forEach( ( fileName ) => {
-			entries[ fileName.replace( /\.[^/.]+$/, '' ) ] = `${ dir }/${ fileName }`;
+			const fullPath = `${ dir }/${ fileName }`;
+			if ( ! fs.lstatSync( fullPath ).isDirectory() ) {
```

Comment:
I've updated this to ignore directories in `assets/css/src`, i.e. only root files in that directory will be used as entries.